README.md

```diff
@@ -2,6 +2,32 @@
 
 Check [prerequisites section](https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/README-prerequisite.md) before you start.
 
+This deployment comprises:
+
+##### Core products
+- Alfresco Process Services 1.10
+- Alfresco Process Services Admin 1.10
+- Alfresco Content Repository 6.1.1
+- Alfresco Content Share 6.1.1
```

Comment:
@flv22  Should this now be 6.2.0 (for both Repo & Share) ?

Note: some of the shared services below such as Transform Router 1.1.0 (and related T-Engines 2.1.0) are aligned & tested with ACS 6.2.0.

Reply:
Yes, currently ACS 6.2 will be pulled, but I'm noticing that requirements are pulling ACS from stable, which I believe should be incubator for the 2.0-SNAPSHOT / develop branch.

@flv22, can you make the changes for this and update the README?